### DISCUSSION TITLE
Sensor info

### DISCUSSION THREAD

**Discussion by eerison** [Nerwyn/universal-remote-card#100] [Q&A] [answered]
Hello guys is there any way to show an sensor info? I would like to show the battery life (0-100%)

**Comment by Nerwyn** [comment, marked as answer]
Not yet, no. As part of the v4 refactor I plan on adding a label option which can be templated to refer to a sensor value ([like my other project](https://github.com/Nerwyn/service-call-tile-feature)).

For now you can change button icons based on sensor values using templates. I discuss and work out changing icon based on battery states with another user in [this issue](https://github.com/Nerwyn/android-tv-card/issues/53#issuecomment-1912624174).

**Reply by eerison** [reply]
Thank you for reply,  change button icon can help me
